### ISSUE TITLE
Will bukkit wrapper switch to remapped-mojang in the future

### ISSUE THREAD

**Issue by MakeDecisionWorth** [webbukkit/dynmap#4170]
I think it's a great idea to use obfuscation mapping in bukkit wrapper, so that developers can port to new version easily without confusion.

As in https://github.com/webbukkit/dynmap/pull/4164#issuecomment-2466582915 , there were misused NMS calls that can be prevented with human readable code.

**Comment by jacob1** [comment]
Agreed. Mistakes are inevitable in the current system. Minor updates are now content updates, meaning most will have breaking changes. Often hard to notice. I think I got them all, but without a comment saying what they were *supposed* to be it's hard to trust even the previous version's mapping as correct.

1.21.4 also has changes you'd never catch, because it compiles (almost) fine with 1.21.3's mappings but it changed other functions behind the scenes.
